dashboard/scripts/archive/restore_level_rename_progress

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+# Add script ids to existing UserLevels. Because in the 'past',
+# UserLevels were independent of script, we don't know what script
+# they belong to, so we'll just create a new one for every possible
+# script that the level is in.
+
+require_relative '../../config/environment'
+
+course_titles_to_renames = {
+    'course1' => {
+        'Course1MazeAssessment1' => 'K-1 Maze Sequence Assessment 1',
+        'Course1MazeAssessment2' => 'K-1 Maze Sequence Assessment 2',
+        'Course1BeePuzzle12' => 'K-1 Bee Puzzle 12',
+        'course 1 artist 1 level 9' => 'K-1 artist 1 level 9',
+        'course 1 artist 1 level 10' => 'K-1 artist 1 level 10',
+        'Building a Foundation Assessment' => 'K-1 Building a Foundation Assessment'
+    },
+    'course2' => {
+        'Course 2 Maze Loops 1' => '2-3 Maze Loops 1',
+        'Course 2 Bee Loops 1' => '2-3 Bee Loops 1A',
+        'Course 2 Bee Loops 2' => '2-3 Bee Loops 2A',
+        'Course 2 Bee Loops 3_' => '2-3 Bee Loops 3A',
+        'Course 2 Bee Loops 3' => '2-3 Bee Loops 3B',
+        'Course 2 Bee Loops 4' => '2-3 Bee Loops 4A',
+        'Course 2 Bee Loops 13' => '2-3 Bee Loops 13'
+    },
+    'course3' => {
+        'Artist - Nested Loops 1' => '4-5 Nested Loops 1',
+        'Artist - Nested Loops 2' => '4-5 Nested Loops 2',
+        'Artist - Nested Loops 6' => '4-5 Nested Loops 6',
+        'Artist - Nested Loops 9' => '4-5 Nested Loops 9',
+        'Artist - Nested Loops 3' => '4-5 Nested Loops 3',
+        'Artist - Nested Loops 5' => '4-5 Nested Loops 5',
+        'Artist - Nested Loops 7' => '4-5 Nested Loops 7',
+        'Artist - Nested Loops 11' => '4-5 Nested Loops 11',
+        'Artist - Nested Loops 8' => '4-5 Nested Loops 8',
+        'Artist - Nested Loops 12' => '4-5 Nested Loops 12',
+        'Artist - Nested Loops Assessment 1' => '4-5 Nested Loops Assessment 1',
+        'C3-Farmer While Loops 1' => '4-5 While Loops 1',
+        'C3-Farmer While Loops 2' => '4-5 While Loops 2',
+        'C3-Farmer While Loops 3' => '4-5 While Loops 3',
+        'C3-Farmer While Loops 4' => '4-5 While Loops 4',
+        'C3-Farmer While Loops 5' => '4-5 While Loops 5',
+        'C3-Farmer While Loops 6' => '4-5 While Loops 6',
+        'C3-Farmer While Loops 7' => '4-5 While Loops 7',
+        'C3-Farmer While Loops 8' => '4-5 While Loops 8',
+        'C3-Farmer While Loops Assessment 1' => '4-5 While Loops Assessment 1'
```

Comment:
Tanya said no need to restore `course4pre` usages, not in active use at the moment